scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobEmailNotification.java

```diff
@@ -72,9 +72,8 @@ public boolean doCheckAndSend() throws JobEmailNotificationException {
             logger.debug("Notification emails disabled, doing nothing");
             return false;
         }
-        if (jobStatusList != null && jobStatusList.contains(eventType.toString().toLowerCase())) {
-            logger.debug("Event type is not mentioned to be noticed, doing nothing");
-            return true;
+        if(jobStatusList!=null && !jobStatusList.contains(eventType.toString().toLowerCase())){
```

Comment:
Yes, it is necessary, if we don't check here, it will give a null pointer exception in the test, because the generic information can be empty